.travis.yml

```diff
@@ -1,21 +0,0 @@
-language: node_js
-dist: trusty
-sudo: required
-node_js: stable
-env:
-  - PATH=$HOME/purescript:$PATH
-install:
-  - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
-  - wget -O $HOME/purescript.tar.gz https://github.com/purescript/purescript/releases/download/$TAG/linux64.tar.gz
-  - tar -xvf $HOME/purescript.tar.gz -C $HOME/
-  - chmod a+x $HOME/purescript
-  - npm install -g bower
-  - npm install
-script:
-  - bower install
-  - npm run -s build
-after_success:
-- >-
```

Comment:
No tests were run here...

Reply:
@kl0tl enabled the tests via GitHub actions in https://github.com/purescript-web/purescript-web-dom-xpath/pull/6/commits/ada7d6e11e622fc740a67d6fdd6ae41806b6bf08